update hazelcast to 5.1.1 (#143)
OpenLiberty/draft-guide-sessions: README.adoc

include::finish/src/main/liberty/config/hazelcast-config.xml[]
----

The cluster name [hotspot=cartCluster file=1]`CartCluster` is defined in the [hotspot file=1]`hazelcast-config.xml`.

In the [hotspot file=0]`server.xml` file, a reference to the Hazelcast configuration file is made by using
the [hotspot=httpSessionCache file=0]`httpSessionCache` tag.

[role="code_command hotspot file=2", subs="quotes"]
----
#Create the `bootstrap.properties` file.#
`src/main/liberty/config/bootstrap.properties`
----

bootstrap.properties
[source, text, linenums, indent=0, role="code_column"]
----
include::finish/src/main/liberty/config/bootstrap.properties[]
----

> Hazelcast JCache provides the client and member providers. Set `hazelcast.jcache.provider.type` to `member` for using the member provider.

Reviewer: ```suggestion
Hazelcast JCache provides the client and member providers. Set `hazelcast.jcache.provider.type` to `member` to use the member provider.
```

a Kubernetes resource definition, which is provided in the [hotspot file=0]`kubernetes.yaml` file
at the `start` directory.

> First, use `ClusterRoleBinding` Kubernetes API object to grant Hazelcast members to access the cluster.

Reviewer: ```suggestion
First, use  the `ClusterRoleBinding` Kubernetes API object to grant Hazelcast members to access the cluster.
```

== Tearing down the environment

> When you no longer need your deployed application, you can delete all {kube} resources and disable the Hazelcast members to access the cluster by running the `kubectl delete` commands:

Reviewer: ```suggestion
When you no longer need your deployed application, you can delete all {kube} resources and disable the Hazelcast members' access to the cluster by running the `kubectl delete` commands:
```
